ci/vars-zuul-common.yml

```diff
@@ -5,3 +5,8 @@ setup_bundle_registry_auth: false
 __service_telemetry_transports_qdr_auth: none
 base_dir: "{{ sto_dir }}/build"
```

Comment:
Where is sto_dir defined for Zuul?

Reply:
When zuul runs, the entrypoint is deploy_stf.yml, which sets sto_dir right away[1] before importing the variables in this file.

[1] https://github.com/infrawatch/service-telemetry-operator/pull/512/files#diff-2873fa0b1fe3441a27c4eead4f6fb7a81095410bbfdfdb7f011526e9f555fa9cR7